src/parser/__init__.py

```diff
@@ -7,180 +8,215 @@
 import sys
 import importlib
 from bs4 import BeautifulSoup
+from typing import Tuple, List, Optional
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+parent_dir = os.path.dirname(current_dir)
+project_root = os.path.dirname(parent_dir)
 
 class Parser:
 
     def __init__(self):
         pass
 
-    def get_class_and_method(self, county):
+    def configure_logger(self):
+        logger = logging.getLogger(name="pid: " + str(os.getpid()))
+        logging.basicConfig(
+            level=logging.INFO,
+            format='%(asctime)s - %(levelname)s - %(message)s',
+            handlers=[
+                logging.FileHandler("parser_log.txt"),
+                logging.StreamHandler() 
+            ]
+        )
+        logger.info("Logger configured")
+        return logger
+    
+    def get_class_and_method(self, logger, county: str, test = False) -> Tuple[Optional[object], Optional[callable]]:
```

Comment:
Just a small thing and nothing to do on this, but I think we should note that get_class_and_method should return an object (the class) and callable (the method)--ie. not optional--or else I'd consider it an error.